CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Contributing Guidelines
+
+You can contribute to the University of Adelaide Computer Science Club (CS Club) website with issues, PRs, and testing of PRs. We appreciate your efforts to make our website better.
+
+## Reporting Issues 
+
+We always welcome bug reports, feature requests and overall feedback. Here are a few tips on how you can make an effective issue report.
+
+### Finding Existing Issues
+
+Before creating a new issue, please search our [open issues](https://github.com/compsci-adl/website/issues) to check if it already exists. If you do find an existing issue, please add your own feedback in the discussion.
+
+### Writing a Good Bug Report
+
+Writing good bug reports make it easier for maintainers to verify and identify the root cause of the problem. The better the bug report, the faster the problem can be resolved. A bug report should contain the following information:
+
+- **Description of the Bug**: What's the issue you encountered?
+- **Reproduction Steps**: How can the issue be reproduced?
+- **Expected Behaviour**: What do you expect to happen?
+- **Information on the environment**: OS, Browser, Device
+- **Additional Notes**: Add any other notes or screenshots about the bug here.
+
+When you are ready to submit a bug report, please use the [Bug Report issue template](https://github.com/compsci-adl/website/issues/new?assignees=&labels=&projects=&template=bug-report.yml).
+
+### Writing a Good Feature Request
+
+Writing good feature requests make it easier for maintainers to understand the feature and see how it fits into the website. The better the feature request, the faster the feature can be discussed and possibly implemented. A feature request should contain the following information:
+
+- **Overview**: Include the basic, high-level concepts for this feature here.
+- **Details**: These may include specific methods of implementation, design considerations, or any other technical details.
+- **Why would this feature be useful?**: A clear and concise description of why this feature would improve the CS Club website.
+- **Additional Notes**: Add any other notes or screenshots about the feature request here.
+
+When you are ready to submit a request, please use the [Feature Request issue template](https://github.com/compsci-adl/website/issues/new?assignees=&labels=&projects=&template=feature-request.yml).
+
+## Contributing Changes
+
+Project maintainers will merge changes that both improve the project and meet our code quality standards. 
+
+### Suggested Workflow
+
+We recommend you to use the following workflow:
+
+1. Fork and clone the repo to your computer.
+```bash
+git clone https://github.com/YOUR_GITHUB_USERNAME/website.git
```

Comment:
sry this is a mistake in my projext
```suggestion
git clone git@github.com:compsci-adl/website.git
```

Reply:
I agree that using ssh is better than http, especially for private repos and when committing, I think http is just easier for beginners cloning a public repository. Because with ssh, you'd need to set up ssh keys and whatnot.